.eslintrc.json

```diff
@@ -60,6 +60,21 @@
     ],
     "no-irregular-whitespace": [
       "error"
+    ],
+    "no-console": [
```

Comment:
This rule is to strict, imo. Sometimes we add experimental code and logging infos to the consoles should not result in a linter error.